src/main/resources/default/assets/scripts/smart-values.js

```diff
@@ -30,11 +30,15 @@ function openSmartValues(elementId, type, payload, signature) {
         securityHash: signature
     }).then(json => {
         if (json.values.length === 0) {
-            _element.tooltip.hide();
-            _element.classList.add('text-decoration-none');
-            _element.classList.remove('link');
-            _element.href = '';
-            return;
+            if(tenantId === "") {
+                _element.tooltip.hide();
+                _element.classList.add('text-decoration-none');
+                _element.classList.remove('link');
+                _element.href = '';
+                return;
+            }else{
+                json.values ={icon:'fa-solid fa-pen-to-square', label:editorLabel, action: '/tenant/' + tenantId};
```

Comment:
Required: Please format the file... the indetion does not seem correct, especially for the `else` and the `json.values`